README.md

```diff
@@ -562,6 +563,80 @@ if ($payment instanceof PaymentExecutedInterface || $payment instanceof PaymentS
 }
 ```
 
+<a name="refunds"></a>
+
+# Refunds
+
+Refunds are only supported for settled merchant account payments.
+
+## Creating and retrieving refunds from the client
+
+```php
+use TrueLayer\Interfaces\Payment\RefundRetrievedInterface;
+use TrueLayer\Interfaces\Payment\RefundExecutedInterface;
+use TrueLayer\Interfaces\Payment\RefundFailedInterface;
+
+// Create and get the refund id
+$refundId = $client->refund()
+    ->payment($paymentId) // Payment ID, PaymentRetrievedInterface or PaymentCreatedInterface
+    ->amountInMinor(1)
```

Comment:
What if you wanted to refund the whole amount? You can omit setting the amount right?

Reply:
Fair enough. Just that in our API, the amount in minor is optional: https://docs.truelayer.com/docs/refunds#make-a-full-or-a-partial-refund-on-a-single-payment

Reply:
It's a good point! I'll have a think about it!